packages/core/android/build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
   }
 
   dependencies {
-    classpath "com.android.tools.build:gradle:7.2.1"
+    classpath "com.android.tools.build:gradle:7.4.2"
```

Comment:
I think this update should also be extended to `react-native-tracer-provider` and `react-native-otlp` packages, right?

and maybe we can add a mention to this in NATIVE_MODULE_DEVELOPING.md, since the `create-react-native-library` by default implements the `com.android.tools.build:gradle:7.2.1` and `https\://services.gradle.org/distributions/gradle-7.4.2-bin.zip`

Reply:
yep makes sense, updated the packages and the documentation